test-support/picolibc-lit-test-constructor-skip.template

```diff
@@ -0,0 +1,4 @@
+# Generated Picolibc lit for @variant@ @picolibc_SOURCE_DIR@/test/@test@.c
+# RUN: %clang @BUILD_PARAMS@
+# RUN: not @TEST_COMMAND@
+# CHECK-OUTPUT: errorcode=0
```

Comment:
This line doesn't do anything.
Is there any output from running the test successfully that you could check against?